api.json

```diff
@@ -0,0 +1,653 @@
+                                    "$ref": "#/components/schemas/Error"
+                                },
+                                "examples": {
+                                    "invalid block level": {
+                                        "$ref": "#/components/examples/invalid-block-level-error"
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/chain/blocks/head": {
+            "get": {
+                "description": "Returns the last applied block.",
+                "tags": [
+                    "chain"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Current block retrieved.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Block"
+                                },
+                                "examples": {
+                                    "block not found": {
+                                        "$ref": "#/components/examples/block-not-found-error"
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/chain/blocks/genesis": {
+            "get": {
+                "description": "Returns the genesis block of the chain.",
+                "tags": [
+                    "chain"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Genesis block has been retrieved.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Block"
+                                },
+                                "examples": {
+                                    "block not found": {
+                                        "$ref": "#/components/examples/block-not-found-error"
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/consensus": {
+            "post": {
+                "description": "nodes related stuff, should only be used by nodes",
+                "tags": [
+                    "consensus"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Consensus operation accepted."
+                    }
+                }
+            }
+        },
+        "/consensus/operations": {
+            "post": {
```

Comment:
- could you define `requestBody` for post operation?
- Is this also the endpoint used for customVM (replacement of user-operation-gossip)?

Reply:
yes, the  `/consensus/operations` is the endpoint to send an operation to the node from a user. Then the node will broadcast it to other nodes